tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -278,7 +278,9 @@ Reference
       | ``[p {options}]{text}[/p]``
 
   * - | **br**
-      | Adds line break in a text.
+      | Adds line break in a text. If used within a list, this won't
```

Comment:
```suggestion
      | Adds line break in a text, without adding a new paragraph. If used within a list, this won't
```

`[br]` will add a break, but it is similar to automatic line breaks added to fit control width, not a new paragraph which is processed independently and can have different properties (alignment, direction, language).